src/components/common/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ id, name, type, textContent, onClick }) => (
+  <button id={id} className='auth-button' name={name} type={type} onClick={onClick}>
```

Comment:
for common componets better not to give such specific classes beacause it seems that this button should only be used for auth. Therefore, if this component is intented to be used only for auth that would be better to rename component to `AuthButton` or change the class to `general-button` or somewhat else and make it possible to pass additional classes as props.